python/llm/README.md

```diff
@@ -39,7 +39,7 @@ Here is an example to use `llm-convert` command line tool.
 # pth model
 llm-convert "/path/to/llama-7b-hf/" --model-format pth --outfile "/path/to/llama-7b-int4/" --model-family "llama"
 # gptq model
```

Comment:
Add link https://huggingface.co/TheBloke/vicuna-13B-1.1-GPTQ-4bit-128g/tree/main  here?

Reply:
@Oscilloscope98 pls help me this?

Reply:
done :)